nautobot/docs/release-notes/version-2.1.md

```diff
@@ -0,0 +1,14 @@
+# Nautobot v2.1
+
+This document describes all new features and changes in Nautobot 2.1.
+
+## Release Overview
+
+### Added
```

Comment:
We need to add the other significant 2.1 features already merged into the release overview:

- Bootstrap/CSS changes
- Admin log view
- isnull filters

Reply:
#4734, #4735, #4736